### DISCUSSION TITLE
Running multiple commands via Windows Powershell

### DISCUSSION THREAD

**Discussion by DOGGOWoo** [4JX/L5P-Keyboard-RGB#94] [Q&A] [answered]
Firstly, I would like to say, this is a great tool. When I first opened Vantage and saw only a limited number of options, I was kind of disappointed. So for the first week or so, I just used my laptop without any backlight. Now however, I'm having quite a blast with the AmbientLight Preset, and wanted to auto start it on startup.

The problem I'm running into is that I want the GUI to be accessible. I have tried doing ".\legion-kb-rgb.exe set -e AmbientLight -s 4 -b 2 gui --hidden" but that just returns "error: Found argument 'gui' which wasn't expected, or isn't valid in this context". Is there any way to have the GUI be accessible while also allowing me to set AmbientLight on startup?

**Comment by 4JX** [comment, marked as answer]
You can save a profile and name it default.json which will get loaded when the program is ran in GUI mode. Though I really should make it possible to specify options *and* launch the GUI at the same time.

**Reply by DOGGOWoo** [reply]
Alright, that seems like a good enough solution for the time being. However, I had another question. Would it be possible to modify the code to make it turn off the keyboard lights when the display has turned off?
